### ISSUE TITLE
Exit button from options has no text

### ISSUE THREAD

**Issue by mikhailnov** [manatools/dnfdragora#188]
Button for exiting from dnfdragora options has no text. dnfdragora-gtk

![2021-04-04_12-15](https://user-images.githubusercontent.com/15802528/113504235-70a0d980-953f-11eb-8ae6-4ff3a0a22232.png)

**Comment by anaselli** [comment]
could be that a translation issue?

![immagine](https://user-images.githubusercontent.com/1303486/113505531-ae056700-953f-11eb-8aa1-784dccaf6cee.png)

**Comment by yurchor** [comment]
@anaselli Might be the echo of the previous issue. The label can be shown when the window width expanded.

**Comment by anaselli** [comment]
@yurchor yeah, i suspected that too ;) I have to understand if using new [YLabel autowrap](https://github.com/libyui/libyui/blob/master/libyui/src/YLabel.h#L144) property for that or force the new line... will check if i can avoid breaking translation first...

**Comment by anaselli** [comment]
Hmm though there was a libyui-gtk issue i fixed and ported on mageia 8 before it was released, gtk windows were not shown well at startup... maybe it's that... even if a new line is a good idea anyway
